keyboards/handwired/scottokeebs/scottonum/readme.md

```diff
@@ -0,0 +1,27 @@
+# ScottoNum
+
+![ScottoNum](https://i.imgur.com/VMiA19uh.jpg)
+
+A 17-key wireless macropad. Case files available [here](https://github.com/joe-scotto/scottokeebs).
```

Comment:
"Wireless" pro micro?

Reply:
Was a leftover from the ZMK version of the firmware, removed "wireless" in [3a25532](https://github.com/qmk/qmk_firmware/pull/21208/commits/3a255324df12b005fbfdbcfb6d4ab9d412e6e3c8).